src/Controls/src/Core/PublicAPI/net-android/PublicAPI.Unshipped.txt

```diff
@@ -1,4 +1,5 @@
 #nullable enable
+Microsoft.Maui.Controls.TabbedPage.TabReselected -> System.EventHandler
```

Comment:
Similar to https://github.com/dotnet/maui/pull/30446#pullrequestreview-2992255760

TabReselected not indicate directly if it's about tapping the already active tab. Because the selection is "just" tapping, not using the keyboard or programmatically, and also implicates the active tab, what about something like TabActiveTapped?